TABTestKit/Classes/Biometrics/Biometrics.h

```diff
@@ -0,0 +1,22 @@
+//
+//  Biometrics.h
+//  TABTestKit
+//
+//  Created by Kane Cheshire on 31/10/2018.
+//  Copyright © 2018 kane.codes. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Biometrics : NSObject
+
++ (void)enrolled;
```

Comment:
We should add some short docs for these public functions

Reply:
Done!